source/renderer/app/components/status/DaedalusDiagnostics.js

```diff
@@ -949,7 +943,7 @@ export default class DaedalusDiagnostics extends Component<Props, State> {
   };
 
   checkTime = () => {
-    this.props.onForceCheckLocalTimeDifference();
+    // this.props.onForceCheckLocalTimeDifference();
```

Comment:
@disassembler can we remove all `onForceCheckLocalTimeDifference` declarations even as comments?

Reply:
@tomothespian we could but there is no time...